share/doc/wake/quickref.md

```diff
@@ -251,26 +251,50 @@ Environment packages can be added to a workspace to provide tools (using runners
 
 #### Sources and `Path` objects
 
-Sources are the set of files in a Git repository. A `Path` in Wake is either a) a file in Git or b) a file produced from a build step. The `Path` type has no public constructor to help enforce this. To get a `Path` value, use the
+Source files are files in a Git repository that are either committed or staged. A `Path` in Wake is either
+
+* a source file or
+* a file produced from a build step.
+
+The `Path` type has no public constructor function to help enforce this. To get a `Path` value for a source file, use the
 
 * `source: (file: String) => Result Path Error`
 * `sources: (dir: String) => (filterRegexp: RegExp) => Result (List Path) Error`
 
-functions. For example, to get all the header files in the current repository inside a Wake source file located at the workspace root:
+functions. For example, to get all the header files under the current Wake root:
 
 ```wake
-sources @here `.*\.h`
+sources "." `.*?\.h`
```

Comment:
I'm not sure you want the question mark (`*` already includes the zero case), but it otherwise looks good!

Reply:
Thanks! I'm not entirely sure why I snuck that in there. I think I got confused with a non-eager case I was thinking about for something else. Nice catch. [Fixed here](https://github.com/sifive/wake/pull/1009/commits/af33fda01701bff9751dd5c5aa550041dc251012).